WindowsServiceManager/WindowsServiceManagerV5/DeployWindowsService.ps1

```diff
@@ -154,10 +154,19 @@ $scriptBlock = {
             $serviceObject = Stop-WSMWindowsService -ServiceName $ServiceName
             Write-Output "[$env:ComputerName]: Deleting the service [$ServiceName]"
             
+            try {
                 $deleteResults = $serviceObject.Delete()
```

Comment:
Is this a terminating error that will get caught?

Reply:
I think yes it was. The Issue was that the Error wasn't good enough to understand what is happening